.changeset/rotten-rats-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+"@chakra-ui/layout": minor
+"@chakra-ui/layout": minor
```

Comment:
I think there is a small typo:

```
"@chakra-ui/theme": minor
```

Reply:
Fixed now, sorry about that, pasted the wrong thing xD